gulpfile.js

```diff
@@ -22,7 +22,22 @@ gulp.task('compile-server', () => {
         .pipe(tsProject())
 
     return tsResult.js
-        .pipe(babel())
+        .pipe(babel({
+            "presets": [
+                "es2015",
+                "stage-3",
+                "react"
+            ],
+            "plugins": [
+                "react-relay",
+                ["module-resolver", {
+                    "alias": {
+                        "components": "./dist/components"
```

Comment:
Do you have a real understanding of how the module resolver plugin was failing and how this fixes it?